.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Should these settings be shared across dev working on this repo? If no then we can remove this file

Reply:
Forgot to add this in the `.gitignore` 🤦🏼‍♂️

Reply:
Even though .vscode is added in .gitignore, still .vscode folder is not removed